### PR TITLE
Link colour contrast fix

### LINKED ISSUE #2: Link colour (grey) is just too light

### PR DESCRIPTION
**Pull request by nowarkdev**
## Fixes
#2 
#103 

## What this PR resolves
Link colours now AA compliant
Link markup on styleguide now uses correct ARIA attributes

## Screenshot before PR (if applicable):
screen shot 2016-02-23 at 14 45 22
screen shot 2016-02-23 at 14 50 22

## Screenshot after PR:
screen shot 2016-02-23 at 14 48 54
screen shot 2016-02-23 at 14 49 27

### CODE REVIEW

File: server/partials/buttons-links.hbs [hunk 1]
```diff
@@ -233,4 +233,4 @@
 <h2 class="h-demo" id="text-links">Text Links</h2>
 
 <a href="#" class="now-link">Standard Link</a>
-<span class="now-link disabled">Disabled Link</span>
+<a href="#" class="now-link disabled" role="button" aria-disabled="true">Disabled Link</a>
```
Comment (peter-mouland) [inline server/partials/buttons-links.hbs]: just to confirm, is `role='button'` being used because there is no href?

Comment (nowarkdev) [inline server/partials/buttons-links.hbs]: @alexnaish @peterthompson we agree, we left it in hoping to spark that discussion. Would be good to know if anyone is using it.

Comment (peter-mouland) [inline server/partials/buttons-links.hbs]: it is in the style-guide... but i think it's a case of over-active designers :)

**Comment by rogerfranklin2** [comment]
Assuming this has been ok'ed by design :+1:

**Comment by nowarkdev** [comment]
Raised with Jamie Green. Will provide feedback when we get it.

**Comment by nowarkdev** [comment]
This has been okayed with Jamie, and will be updated in the UI style guide.

@peterthompson

**Comment by peter-mouland** [comment]
:+1:

**Comment by nowarkdev** [comment]
Possibility that we could update the disabled link colour. Still discussing this with Jamie so will leave this branch for now.

**Pull request merged**